.github/workflows/asana-release.yml

```diff
@@ -3,11 +3,22 @@ name: Autofill Release Workflow
 on:
     release:
         types: [created]
+    workflow_dispatch:
+        inputs:
+            tag_name:
+                description: 'Tag name for the release'
+                required: true
+            release_url:
+                description: 'URL for the release'
+                required: true
+            release_notes:
+                description: 'Release notes for the release'
+                required: true
 
 env:
-    VERSION: ${{ github.event.release.tag_name }}
-    RELEASE_URL: ${{ github.event.release.html_url }}
-    RELEASE_NOTES: ${{ github.event.release.body }}
+    VERSION: ${{ github.event.inputs.tag_name || github.event.release.tag_name }}
+    RELEASE_URL: ${{ github.event.inputs.release_url || github.event.release.html_url }}
+    RELEASE_NOTES: ${{ github.event.inputs.release_notes || github.event.release.body }}
```

Comment:
This was needed to allow for `workflow_dispatch` support - now the input values will get used if they exist, otherwise it'll fall back to the GitHub release values. This change is also the reason for replacing a bunch of `github.event` usage below with `env.VERSION`.